acl-lib/src/cmr/acl/core.clj

```diff
@@ -130,15 +135,23 @@
   * target = \"INGEST_MANAGEMENT_ACL\"
   * permission-type = :read"
   [context object-identity-type target permission-type]
-  (let [acl-oit-key (echo-acls/acl-type->acl-key object-identity-type)]
-    (->> (acl-fetcher/get-acls context [object-identity-type])
-         ;; Find acls on INGEST_MANAGEMENT
-         (filter #(= target (get-in % [acl-oit-key :target])))
-         ;; Find acls for this user and permission type
-         (filter (partial acl-matches-sids-and-permission?
-                          (context->sids context)
-                          permission-type))
-         seq)))
+  (try
+    (let [acl-oit-key (echo-acls/acl-type->acl-key object-identity-type)]
+      (->> (acl-fetcher/get-acls context [object-identity-type])
+           ;; Find acls on INGEST_MANAGEMENT
+           (filter #(= target (get-in % [acl-oit-key :target])))
+           ;; Find acls for this user and permission type
+           (filter (partial acl-matches-sids-and-permission?
+                            (context->sids context)
+                            permission-type))
+           seq))
+    (catch Exception e
+      (if (re-matches #".*status 401.*" (.getMessage e))
+        (let [error-message (peek (re-find #"\[\"(.+?)\"" (.getMessage e)))]
+          (errors/throw-service-error
+           :unauthorized
+           error-message))
+        nil))))
```

Comment:
Are there other exceptions that could occur here that we are just throwing away?